src/style/index.scss

```
@@ -117,14 +117,102 @@ hr.spacer {
+    .fees-details-shell {
+        transition: none;
+    }
+
+    .fees-details-inner {
+        transition: none;
+        opacity: 1;
+    }
+}
+
+.fees-abbrev-underline {
+    border-bottom: 1px dashed currentColor;
```

Stylelint (17.5.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)